Integrations/AutofocusV2/AutofocusV2.py

```diff
@@ -973,6 +977,66 @@ def get_indicator_outputs(indicator_type, indicators, indicator_context_output):
     return_outputs(readable_output=human_readable, outputs=ec, raw_response=raw_res)
 
 
+def check_for_ip(indicator):
```

Comment:
shouldn't it be in commonserverpython?

Reply:
we can add this... but it might be specific to this implementation (it is taken from the miner itself with some minor modifications)

Reply:
i think that we should have something for that in the commonserverpython, @bakatzir maybe you already done something for this?

Reply:
+1
I haven't since office365 can also have urls so it's a bit different.
but we should implement it in commonserverpython using the enums from this pr:
https://github.com/demisto/content/pull/5368

Reply:
While I think we should definitely move something like this to commonserverpython, this uses the `ipaddress` module which comes only with python3. If we move to commonserverpython we should make sure it's 2&3 compatible.